tests/test_components.py

```diff
@@ -615,7 +655,28 @@ def test_1d_output_matches_cached_output(self):
         assert True
 
 
-def testDcmipOptions():
+def test_dcmip_validate_inputs():
+
+    dcmip = DcmipInitialConditions()
+
+    state = climt.get_default_state([dcmip],
+                                    y=dict(label='latitude',
+                                           values=np.linspace(0, 60, 20),
+                                           units='degrees_north'))
+
+    with pytest.raises(ValueError) as excinfo:
+        dcmip(state, type_of_output='abcd')
+
+    assert 'has to be one' in str(excinfo.value)
```

Comment:
Does this code work? I'd think this should be indented.

Reply:
yup, this is the syntax to test for any Error raised.